test/e2e/app-dir/weird-chars-in-scripts/weird-chars-in-scripts.test.ts

```diff
@@ -2,12 +2,12 @@
 import { nextTestSetup } from 'e2e-utils'
 
 describe('weird chars in scripts', () => {
-  const { next } = nextTestSetup({
+  const { next, isNextStart } = nextTestSetup({
     files: __dirname,
   })
   // TODO: fix test case in webpack
   // It's failing with `Could not find the module ".../app/client#component.tsx#" in the React Client Manifest. This is probably a bug in the React Server Components bundler.`
-  ;(process.env.TURBOPACK ? it : it.skip)(
+  ;(process.env.TURBOPACK && !isNextStart ? it : it.skip)(
```

Comment:
If that test case is failing for next build --turbo, but it's intended to work, better add it as failing test to the test manifest, so we keep tracking it.
Otherwise it's very easy to forget about that

Reply:
Did it